Softeq.XToolkit.WhiteLabel.Droid/Navigation/StartActivity.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace Softeq.XToolkit.WhiteLabel.Droid.Navigation
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public class StartActivity : Attribute
```

Comment:
I think need to rename to `StartActivityAttribute` because `StartActivity` is confusing.
Maybe rename to `RootActivityAttribute`.